JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -126,6 +133,11 @@ module Microsoft.ApplicationInsights {
                 }
 
                 this._pageTracking.stop(name, url, properties, measurements);
+
+                if (this.config.autoTrackPageVisitTime) {
+                    this.autoTrackPageVisitTime(name, url);
```

Comment:
Why are we using a _custom metric_ to track page view duration in addition to the _page view_ telemetry already tracked with duration here?